config/webpack.common.js

```diff
@@ -27,6 +28,13 @@ module.exports = {
         exclude: /(node_modules)/,
         use: ['babel-loader', 'eslint-loader']
       },
+      {
+        test: /\.s?css$/i,
+        use: ExtractTextPlugin.extract({
```

Comment:
Did you check what is more performant in production? Having our styles in our html file or as separate files?

Reply:
It makes it easy to split css into multiple files in situations they become large(lazy loading)

Reply:
Niceee. I'd like to see a usage of that when we start building. Thanks